Chaincase.UI/Pages/NewPasswordPage.razor

```diff
@@ -11,25 +11,48 @@
 @inherits ReactiveComponentBase<NewPasswordViewModel>
 
 <ion-content class="ion-padding">
-    <h1>Set your password</h1>
-    <IonItem>
-        <ion-label floating>Password</ion-label>
-        <IonPasswordInput @bind-Value="@Password" placeholder="e.g. margaretthatcheris110%SEXY" />
-    </IonItem>
-    <IonButton OnClick='SubmitPassword' expand="block">NEXT</IonButton>
+    <EditForm EditContext="_editContext" OnSubmit="SubmitPassword">
+        <h1>Set your password</h1>
+        <IonItem ValidationField="() => Password">
+            <ion-label floating>Password</ion-label>
+             <IonPasswordInput clear-on-edit="false" @bind-Value="@Password" placeholder="e.g. margaretthatcheris110%SEXY" />
```

Comment:
the only difference between this and #105 here but the validation works without it
 ```html
 <IonPasswordInput ValidationField="() => Password" clear-on-edit="false" @bind-Value="@Password" placeholder="e.g. margaretthatcheris110%SEXY"/>
```